wavfile24.py

```
@@ -0,0 +1,294 @@
+    noc = 1
+    bits = 8
+    #_cue = []
+    #_cuelabels = []
+    _markersdict = collections.defaultdict(lambda: {'position': -1, 'label': ''})
+    loops = []
+    pitch = 0.0
+    while (fid.tell() < fsize):
+        # read the next chunk
+        chunk_id = fid.read(4)
+        if chunk_id == b'fmt ':
+            size, comp, noc, rate, sbytes, ba, bits = _read_fmt_chunk(fid)
```

Ruff (0.13.3)

Unpacked variable `comp` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)